tests/python/unittest/test_meta_schedule_tune_relay.py

```diff
@@ -144,14 +144,19 @@ def test_meta_schedule_tune_relay(
             mod=mod,
             params=params,
             target=target,
-            config=ReplayTraceConfig(
+            config=TuneConfig(
+                strategy="evolutionary",
                 num_trials_per_iter=32,
-                max_trials_per_task=32,
+                max_trials_per_task=20000,
```

Comment:
In general, do you recommend setting `max_trials_per_task` to be the same value as `max_trials_global`, or some arbitrary large number? This change makes me think that the value of `max_trials_per_task` doesn't matter much as long as it is sufficiently big.

Reply:
I see now that the value of `max_trials_per_task` is optional:

```
max_trials_per_task: Optional[int] = None
```

How about removing the explicit specification of ` max_trials_per_task` here and elsewhere in tests, if this value is not important?

Reply:
In this case, I think `max_trials_global` is the default for `max_trials_per_task`, except when we want to limit the trials for each task. In these cases, if the search space is very small, the evolutionary search strategy would do an early stopping; when we only want to do test, we can limit the `max_trials_global` to be a small number close to `num_trials_per_iter*task_num`. Therefore, I think it could be a good move to use `max_trials_global` as default to `max_trials_per_task` unless specified.

Reply:
yeah I mean, if this value is optional, at least one test should exercise that scenario / code path.

Reply:
Hi Masa, this is actually a great point! Generally speaking, I would recommend `max_trials_per_task 
 == max_trials_global` by default, and leave the TaskSchedule to decide how to allocate those trials

Reply:
Let me improve this API by marking `max_trials_per_task` as optional

Reply:
oh actually I just found myself implemented the logic in `create_strategy`...Let me update the docs to reflect this

Reply:
IMHO we should still set `max_trials_global` as something like `2000` here and add comment `suggested 20k or 1k * num of subgraphs` beside because this is still a functionality test. I will send a PR for meta schedule tutorial as Masa suggested later this week and that would be a better example to show case our interface usage.

Reply:
Well, yes or no. There is no "definitely correct" suggestion in terms of how many trials should be allocated globally or locally...